demos-europe/demosplan-core
# feat(refs DPLAN-11981): Enable images for segments

### Ticket
DPLAN-11981

Add the possibility to insert images in recommendation texts and export them with docx and pdf. 

FE changes:
- Enable image button in editor toolbar
- On click image should open in a modal (in editor with strg + click)
- Add hint for xlsx export

### How to review/test
For (fully) testing locally you need to install the demosplan-ui changes linked below.

Verfahren -> Abschnitte -> Erwiderung verfassen -> Bearbeiten -> Add image
On click should work in editor, after saving editor changes and Verfahren -> Abschnitte 

### Linked PRs (optional)
https://github.com/demos-europe/demosplan-ui/pull/919



### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
